SECURITY.md

```diff
@@ -0,0 +1,21 @@
+# Security Policy
+
+DarkReader is known to have no intentional security vulnerabilities.
+If reported, vulnerabilites will be fixed in the shortest time possible,
```

Comment:
`shortest time and highest priority as possible`

Reply:
Fixing priority is deliberately left out as it would imply a warranty policy.